### PR TITLE
endpoint for querying content status

### PR DESCRIPTION
**Pull request by RadNi**
`queryContent` endpoint and its tests in `test/Test/Trace.hs` implemented. #217

### CODE REVIEW

**Review by zygomeb — APPROVED** [review]

**Comment by cstml** [comment]
@RadNi can you please push your branch to the `mlabs` repo so `CI` can run on it?

File: mlabs/src/Mlabs/NFT/Api.hs [hunk 1]
```diff
@@ -51,18 +64,18 @@ mkSchemaDefinitions ''NFTAppSchema
 -- ENDPOINTS --
 
 type ApiUserContract a = UserContract NFTAppSchema a
-
 type ApiAdminContract a = AdminContract NFTAppSchema a
 
+--type ApiQueryContract a = Contract (Last QueryResponse) NFTAppSchema Text a
+
 -- | User Endpoints .
 endpoints :: NftAppSymbol -> ApiUserContract ()
```
Comment (cstml) [inline mlabs/src/Mlabs/NFT/Api.hs:72]: You will need to merge with the new `staging` structure.

**Comment by RadNi** [comment]
@cstml currently don't have access to push a branch to `mlabs` repo

File: mlabs/src/Mlabs/NFT/Api.hs (hunk 1 above)
Comment (RadNi) [reply mlabs/src/Mlabs/NFT/Api.hs]: @cstml merged

**Comment by cstml** [comment]
Moved to: #247 to get CI

**Comment by cstml** [comment]
> @cstml currently don't have access to push a branch to `mlabs` repo

Ok, I've pinged people to add you. Please lmk if not resolved after today.

**Pull request closed without merging**